src/fr/starboundscans/build.gradle

```diff
@@ -1,7 +1,8 @@
 ext {
     extName = 'Starbound Scans'
     extClass = '.StarboundScans'
-    themePkg = 'keyoapp'
+    extVersionCode = 1
```

Comment:
```suggestion
```

You should modify `overrideVersionCode` so the old version code (Madara + override) < the new version code (Keyoapp + override).